### ISSUE TITLE
Tactical Store: Documented interface exposed for data manager to use.

### ISSUE THREAD

**Issue by alxhub** [angular/tactical#7]
The Tactical Store should have documented operations for retrieving data by key as well as committing a new version of an object.

**Comment by ttowncompiled** [comment]
### Tactical Store API
```typescript
fetch(key: Object): Object | null
```
Returns the object _(in the Tactical Store)_ that is associated with the provided key or &mdash; if no object is associated with the key &mdash; returns null.

**Comment by ttowncompiled** [comment]
### Tactical Store API - usable by _Data Manager_
```typescript
commit(key: Object, value: Object): bool;
```
This will check if an object is associated with the provided key. If an object is, then that object will be overwritten with the provided value. If an object is not, then the provided value will be added to the Tactical Store. This function will return true unless an error is caught. If so, then this function will return false.

**Comment by ttowncompiled** [comment]
Blocked on #21

**Comment by ttowncompiled** [comment]
Solved #23
